doc/12-icinga2-api.md

```diff
@@ -1746,10 +1806,11 @@ must support long-polling and HTTP/1.1. HTTP/1.0 is not supported.
 Example:
 
 ```
-$ curl -k -s -u root:icinga -H 'Accept: application/json' \
+$ curl -k -s -S -i -u root:icinga -H 'Accept: application/json' \
  -X POST 'https://localhost:5665/v1/events' \
  -d '{ "queue": "myqueue", "types": [ "CheckResult" ], "filter": "event.check_result.exit_status==2" }'
```

Comment:
You're missing a <code>```</code> line here.

It might be helpful for you to check if https://github.com/Icinga/icinga2/blob/bugfix/doc-use-S-parameter-8065/doc/12-icinga2-api.md renders correctly.